fusion/java/sample.java

```diff
@@ -0,0 +1,73 @@
+import java.io.IOException;
+import okhttp3.*;
+import okhttp3.Request.Builder;
+import org.json.*;
+
+// using AOuth 2.0 to access Yelp Fusion api
+
+public class sample {
+
+    public static void main(String[] args)throws JSONException, IOException {
+
+
+        // POST https://api.yelp.com/oauth2/token
```

Comment:
using the oauth2 endpoint is no longer necessary, more details [here](https://www.yelp.com/developers/documentation/v3/authentication#where-is-my-client-secret-going) could you delete this section?